lldb/source/Symbol/SwiftASTContext.cpp

```diff
@@ -1448,25 +1448,45 @@ void ApplyWorkingDir(SmallString &clang_argument, StringRef cur_working_dir) {
   llvm::sys::path::append(clang_argument, cur_working_dir, rel_path);
   llvm::sys::path::remove_dots(clang_argument);
 }
+
+enum ClangArgCategory {
+  ClangArgCategoryNone = 0,
+  ClangArgCategoryUnique = 1 << 0,
+  ClangArgCategoryMacro = 1 << 1,
+};
+
```

Comment:
I don't understand the point of this enum. It looks like all its used for is to set two separate bools, and it appears that we could split `GetClangArgCategory` into a static `shouldUnique`  and a `isMacro` function.

Reply:
Maybe this is Swift-inspired deformation, I tend to avoid flag-driven return types when possible. Of course, I won't insist here, it's more important to maintain common practices from neighboring code if you suppose that couple of bool returning functions would look better here.